Python/Product/Debugger/CustomDebugAdapterProtocolExtension.cs

```diff
@@ -87,6 +89,23 @@ public static void SendRequestSync<TArgs>(DebugRequest<TArgs> request)
         public void RegisterCustomMessages(ICustomMessageRegistry registry, IProtocolHostOperations hostOperations) {
             _hostOperations = hostOperations;
 
+            registry.RegisterEventType<VersionInfoEventArgs>(OnVersionInfo);
+            registry.SetInitializeRequestProperty("supportsVersionInfo", true);
+        }
+
+        private void OnVersionInfo(VersionInfoEventArgs e) {
+            if (PackageVersion.TryParse(e.Version, out PackageVersion runningVersion)) {
+                var bundledPtvsdVersionStr = "4.0.0a6";
```

Comment:
At least make it const :) Top of the class is good too (or if there's a convenient Constants.cs file)

Reply:
This was temporary. I am going to update with the proj based change.